src/main/resources/assets/styles/main.scss

```diff
@@ -123,6 +123,11 @@ section.part-divider + section.part-divider {
   background-color: $ssb-dark-5;
   padding-top: 20px;
   padding-bottom: 20px;
+
+  /* Set text color to white for all child elements */
+  & * {
+    color: white;
```

Comment:
Might be better to use `ssb-white` here instead for best practice

Reply:
Endret til ssb-white